
# HG changeset patch
# User Jonathan Griffin <jgriffin@mozilla.com>
# Date 1452103187 28800
# Node ID 92ae6ef920a1904f7742f74514026db21e54fe17
# Parent  1ec3a3ff68f2d1a54e6ed33e926c28fee286bdf1
try: -b do -p win32 -u mochitest-1,mochitest-2,mochitest-3,mochitest-4,mochitest-5 -t none

diff --git a/testing/mozharness/configs/unittests/win_unittest.py b/testing/mozharness/configs/unittests/win_unittest.py
--- a/testing/mozharness/configs/unittests/win_unittest.py
+++ b/testing/mozharness/configs/unittests/win_unittest.py
@@ -78,16 +78,17 @@ config = {
         "mochitest": {
             "options": [
                 "--appname=%(binary_path)s",
                 "--utility-path=tests/bin",
                 "--extra-profile-file=tests/bin/plugins",
                 "--symbols-path=%(symbols_path)s",
                 "--certificate-path=tests/certs",
                 "--quiet",
+                "--e10s",
                 "--log-raw=%(raw_log_file)s",
                 "--log-errorsummary=%(error_summary_file)s",
                 "--screenshot-on-fail",
             ],
             "run_filename": "runtests.py",
             "testsdir": "mochitest"
         },
         "mozbase": {

